src/android/GeoNotification.java

```diff
@@ -21,19 +26,31 @@ public class GeoNotification {
     @Expose public String authorization;
     @Expose public String startTime;
     @Expose public String endTime;
+    @Expose public boolean isLast;
 
     @Expose public Notification notification;
 
+
+
     public GeoNotification() {
     }
 
     public Geofence toGeofence() {
-        return new Geofence.Builder()
-            .setRequestId(id)
-            .setTransitionTypes(transitionType)
-            .setCircularRegion(latitude, longitude, radius)
-            .setLoiteringDelay(loiteringDelay == 0 ? 60 * 60 * 1000 : loiteringDelay)
-            .setExpirationDuration(Long.MAX_VALUE).build();
+        if(transitionType == 1 || transitionType == 2){
```

Comment:
I am not sure what those numbers represents, can we add comments on the non-obvious part of the code